ui/src/version/VersionListTable.js

```diff
@@ -426,7 +426,7 @@ const VersionListTable = ({
                       <EuiLink
                         onClick={() =>
                           navigate(
-                            `/merlin/projects/${activeModel.project_id}/models/${activeModel.id}/versions/${endpoint.version_id}/endpoints/${endpoint.id}`
+                            `/merlin/projects/${activeModel.project_id}/models/${activeModel.id}/versions/${endpoint.version_id}/endpoints/${endpoint.id}/details`
```

Comment:
Why does this need to change? :o

Reply:
Because without `/details` the model version ui page does not automatically set the Configuration table as active even though we already set index props on /details Route ([ref](https://github.com/caraml-dev/merlin/blob/v0.33.0/ui/src/pages/version/VersionDetails.js#L201)).
<img width="500" alt="Screen Shot 2023-10-18 at 17 24 46" src="https://github.com/caraml-dev/merlin/assets/8122852/4c53c0a2-b8e9-4c0c-a50d-26a33ba26232">

Reply:
Ahh ok got it :) Thanks!